apps/webapp/app/send/constants.ts

```diff
@@ -25,3 +25,18 @@ export const sendTabs = [
   { title: 'Receive', href: DappPath.RECEIVE, active: true },
   { title: 'IBC', href: DappPath.IBC, active: false },
 ];
+
+export const sendMetadata: SendMetadataMap = {
+  [DappPath.SEND]: {
+    title: 'Penumbra | Send',
+    description: EduPanel.SENDING_FUNDS,
+  },
+  [DappPath.RECEIVE]: {
+    title: 'Penumbra | Receive',
+    description: EduPanel.SENDING_FUNDS,
+  },
+  [DappPath.IBC]: {
+    title: 'Penumbra | IBC',
+    description: EduPanel.TEMP_FILLER,
+  },
+};
```

Comment:
have one global map somewhere in /apps/webapp that stores the metadata for all pages.

Also create a `<HeadMetadata />` component to replace: 

```
<title>{sendMetadata[pathname].title}</title>
      <meta name='description' content={sendMetadata[pathname].description} />
```

and it should take `DappPath` as a prop and everything else should be handled for it.